OneCommunityGlobal/HighestGoodNetworkApp
# Sheetal M - Add unit test for managingTeamsReducer

# Description
Unit test for managingTeamsReducer

## Related PRS (if any):
No related PRs

## Main changes explained:
- Added a new file under reducers/tests called managingTeamsReducer.test.js
- The list of test cases for managingTeamsReducer
  - return the initial state when no action is specified
  - handles FETCH_PROJECTS_START action by returning status 200
  - handles RECEIVE_TEAMS action by returning status 200
  - handles FETCH_TEAMS_ERROR by returning fetching=false

## How to test:
1. Checkout the current branch
2. Run npm test managingTeamsReducer.test.js
3. Verify all test cases pass without any errors

## Screenshots or videos of changes:
![manging-teams-reducer-test](https://github.com/user-attachments/assets/a4a7ca09-9df5-4934-969d-67edb60748bc)
